content/posts/android-tv-13.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Android TV v0.13
+subtitle: That was quick
+author: Niels van Velzen
+githubusername: nielsvanvelzen
+date: 2022-03-19
+justify: center
+---
+
+<!-- markdownlint-disable MD033 MD036 -->
+
+We're back with another release for Android TV and Fire TV users containing a lot of bugfixes!
+
+<!--more-->
+
+As promised in the previous release post, the time between updates is getting shorter. We aim for future updates to happen significantly more frequently than the two years between 0.11 and 0.12. Today's update brings some new features and greatly improves playback of videos. Let's get started with the player changes.
+
+## Playback changes
+
+Most changes in this release are related to playing video files, the so called playback code. A lot of changes were made, both bugfixes and new features! The full list is, as always, available in the changelog. The most notable changes are listed below.
```

Comment:
Do we have someway to link the changelog before the actual release?

Reply:
Yeah because we now what the relase tag is going to be. There link was already in the changelog section but I've now changed the word to a link in this paragraph.